Stripe/STPBankSelectionViewController.h

```diff
@@ -0,0 +1,69 @@
+//
+//  STPBankSelectionViewController.h
+//  Stripe
+//
+//  Created by David Estes on 8/9/19.
+//  Copyright © 2019 Stripe, Inc. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import <PassKit/PassKit.h>
+#import "STPCoreTableViewController.h"
+#import "STPFPXBankBrand.h"
+#import "STPPaymentConfiguration.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol STPBankSelectionViewControllerDelegate;
+@class STPPaymentMethodParams;
+
+/** This view controller displays a list of banks of the specified type, allowing the user to select one to pay from.
+    Once a bank is selected, it will return a PaymentMethodParams object, which you can use to confirm a PaymentIntent or
+    share the details with your backend.
```

Comment:
We're not directly recommend it, but slight FUD around people passing up a paymentMethodParams object.  wdyt about leaving it at "...confirm a PaymentIntent".

Reply:
I'd like the dev to still access the bank information somehow, and we're not encouraging them to send up the entire PaymentMethodParams object.

Maybe: "Once a bank is selected, it will return a PaymentMethodParams object, which you can use to confirm a PaymentIntent or inspect to obtain details about the selected bank."

Reply:
+1 sounds good to me!